orb/commands/@drupal.yml

```diff
@@ -75,24 +75,47 @@ drupal-docker-build:
       type: string
       default: "web"
       description: "Path to be used as build context for Nginx image."
+    background:
+      type: boolean
+      default: true
+      description: "Run docker build in background."
+    wait:
+      type: boolean
+      default: true
+      description: "Wait for docker build to finish."
   steps:
     - build-docker-image:
         dockerfile: silta/nginx.Dockerfile
         path: <<parameters.nginx_build_context>>
         identifier: nginx
         docker-hash-prefix: v5
+        background: <<parameters.background>>
 
     - build-docker-image:
         dockerfile: silta/php.Dockerfile
         path: "."
         identifier: php
         docker-hash-prefix: v5
+        background: <<parameters.background>>
 
     - build-docker-image:
         dockerfile: silta/shell.Dockerfile
         path: "."
         identifier: shell
         docker-hash-prefix: v6
+        background: <<parameters.background>>
+
+    - run:
+        name: Wait for docker images to be built
+        command: |
+          if [ << parameters.wait >> = true ]; then
```

Comment:
If wait is false, this step still runs but does not really do anything.

To avoid confusion I would remove this if-statement and instead use the builtin `when.condition` key, like in this example from another part of this file: 

    when:
        # Install dev dependencies.
        condition: <<parameters.install-dev-dependencies>>

Reply:
I think i tried this first, but it did not work for a single step. That's why there are those `when` and `unless` for almost identical steps in `drupal-composer-install:`, have to copy the whole `steps` block.